src/client.c

```diff
@@ -119,16 +119,16 @@ int clientSendHandshake(struct client *c)
 	}
 
 /* Decode a response. */
-#define DECODE(LOWER)                                                \
-	{                                                            \
-		int rv;                                              \
-		struct cursor cursor;                                \
-		cursor.p = buffer__cursor(&c->read, 0);              \
-		cursor.cap = buffer__offset(&c->read);               \
-		rv = response_##LOWER##__decode(&cursor, &response); \
-		if (rv != 0) {                                       \
-			return DQLITE_ERROR;                         \
-		}                                                    \
+#define DECODE(LOWER)                                              \
+	{                                                          \
+		int rv;                                            \
+		struct cursor cursor;                              \
+		cursor.p = bufferCursor(&c->read, 0);              \
+		cursor.cap = bufferOffset(&c->read);               \
+		rv = response_##LOWER##Decode(&cursor, &response); \
```

Comment:
Same here, mixed snake_case and camelCase, probably needs to change `LOWER` to `CAPITAL` or something like that.

Reply:
All these kind of ones were just fixed the the last 6 commits

Reply:
Fixed